dump/plugins/interface.py

```diff
@@ -0,0 +1,265 @@
+from sonic_py_common.interface import get_interface_table_name, get_intf_longname, VLAN_SUB_INTERFACE_SEPARATOR
+from dump.match_infra import MatchRequest
+from dump.helper import create_template_dict, handle_error
+from dump.match_helper import fetch_port_oid, fetch_vlan_oid, fetch_lag_oid
+from swsscommon.swsscommon import SonicDBConfig
+from .executor import Executor
+
+
+class Interface(Executor):
+    """
+    Debug Dump Plugin for Interface Module.
+    Interface can be of Ethernet, PortChannel, Loopback, Vlan or SubInterface type
+    Human readable intf string names are supported
+    """
+    ARG_NAME = "intf_name"
+
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+        self.ns = ''
```

Comment:
Can use https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/multi_asic.py#L26 instead of ''

Reply:
Handled #